package.json

```diff
@@ -3,11 +3,14 @@
   "version": "1.7.0",
   "description": "Blazing-fast binary parser builder",
   "main": "dist/binary_parser.js",
+  "dependencies": {
+    "text-encoding": "^0.7.0"
```

Comment:
I think you want `fast-text-encoding` instead? The `text-encoding` package seems to be no longer maintained.